packages/metro/src/lib/contextModuleTemplates.js

```diff
@@ -26,14 +27,19 @@ function createFileMap(
     .forEach(file => {
       let filePath = path.relative(modulePath, file);
 
+      if (os.platform() === 'win32') {
+        filePath = filePath.replaceAll(path.sep, '/');
```

Comment:
Sorry! Just noted that `String.replaceAll` isn't available in our minimum supported version of Node (14.x) — you were good beforehand 😅.

```suggestion
        filePath = filePath.replace(/\\/g, '/');
```